docs/package_firewall/faq.md

```diff
@@ -0,0 +1,56 @@
+# Frequently Asked Questions
+
+## What do I do when a dependency fails analysis?
+
+If a package's installation gets blocked by the firewall, possible malicious
+code execution will have been blocked, so there's no immediate need for action.
+
+Should you determine the policy violation to be a false-positive, you can add an
+exception by navigating to [the firewall overview], selecting the group
+associated with the project, then clicking on the "Allow Package" button in the
+top-right.
+
+[the firewall overview]: https://app.phylum.io/firewall
+
+## Why are some packages violating a policy not being blocked?
```

Comment:
It might just be me, but I stumbled over this heading a bit. An alternative might be, "Why are some policy-violating packages not being blocked?"

Reply:
Your suggestion seems good. I was honestly struggling a bit with this one because I wanted to write it in a way that doesn't sound like the package firewall is doing something wrong, but your suggestion still achieves that.